### ISSUE TITLE
[BUG] Burts of messages "Can't add new command when connection is in closed state

### ISSUE THREAD

**Issue by sousagaspar** [cweijan/vscode-mysql#699]
- OS: Ubuntu 20.04
- Database Type: MySQL 5.7
- Extension Version: v5.8.11

When a statement is taking too long, if one closes the database connection, sometimes we get a burst of the following messages in the output:

```
2022-10-25 14:24:43 Execute sql fail : SELECT column_name "column_name",index_name "index_name",index_type "index_type",non_unique=0 "isUnique" FROM INFORMATION_SCHEMA.STATISTICS WHERE table_schema='xpto' and table_name='example_table';
2022-10-25 14:24:43 Can't add new command when connection is in closed state
2022-10-25 14:24:43 Execute sql fail : SELECT TRIGGER_NAME FROM information_schema.TRIGGERS WHERE TRIGGER_SCHEMA = 'xpto'  AND EVENT_OBJECT_TABLE='example_table' ORDER BY TRIGGER_NAME;
2022-10-25 14:24:43 Can't add new command when connection is in closed state
```

This messages don't stop showing. One needs to restart the vscode.

**Comment by cweijan** [comment]
This is strange, when this error occurs, the connection will be closed, what is the operation that causes this error?

**Comment by cweijan** [comment]
Fixed in version 6.0.0.
